packages/cloud-config/collection.yaml

```diff
@@ -0,0 +1,39 @@
+packages:
+- &base
+  name: defaults
+  category: cloud-config
+  version: "0.1"
+  oem_file: "01_defaults.yaml"
+  templated: false
+  requires:
+  - name: "cos-setup"
+    category: "system"
+    version: ">=0"
+- <<: *base
+  name: "rootfs"
+  oem_file: "00_rootfs.yaml.tmpl"
```

Comment:
SUPER NIT: why is the 00_X file the second one done. This is killing my OCD. can it be put in order?